lambdas/choice/src/writer.py

```diff
@@ -1,6 +1,78 @@
 import os
+import boto3
+
+"""
+Processes SNS messages containing user choices by writing choices to Dynamo.
+For event and database record shapes, see https://github.com/sjawhar/enchanted-brain/issues/4.
+"""
+
+
+EVENT_USER_ID_KEY = 'userId'
+EVENT_CHOICE_TYPE_KEY = 'choiceType'
+EVENT_CHOICE_KEY = 'choice'
+EVENT_TIMESTAMP_KEY = 'timestamp'
+
+CHOICE_COLOR = 'CHOICE_COLOR'
+CHOICE_EMOTION = 'CHOICE_EMOTION'
+CHOICE_CHILLS = 'CHOICE_CHILLS'
+CHOICE_IMAGERY = 'CHOICE_IMAGERY'
+
+CONCERT_SONG_ID = os.environ['EVENT_NAME']
+DYNAMODB_TABLE_NAME = os.environ['DYNAMODB_TABLE_NAME']
+
+choice_type_to_choice_key = {
+    CHOICE_COLOR: 'colors',
+    CHOICE_EMOTION: 'emotions',
+    CHOICE_CHILLS: 'chills',
+    CHOICE_IMAGERY: 'imagery'
+}
+choice_types = choice_type_to_choice_key.keys()
+
+dynamodb = boto3.resource(
+    'dynamodb',
+    region_name=os.environ['AWS_DYNAMO_REGION'],
+    endpoint_url=os.environ['AWS_DYNAMO_ENDPOINT'],
+    aws_secret_access_key=os.environ['AWS_SECRET_ACCESS_KEY'],
+    aws_access_key_id=os.environ['AWS_ACCESS_KEY_ID']
```

Comment:
These credentials are only needed in local development, right?

Reply:
Actually, now that we have the DynamoDB client stubbed properly, I don't think they're needed at all. Removed all except for endpoint, which has been updated per suggestion above.

Reply:
Actually, I think that we can get rid of the two access key parameters entirely. I've updated the other two according to what you suggested for DYNAMODB_ENDPOINT above.